tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php

```diff
@@ -128,9 +128,10 @@ public static function deleteDummyOrderByIncrementId($incrementId)
         $resource = Mage::getSingleton('core/resource');
```

Comment:
@mccottry the same with order, you could use 
Mage::register('isSecureArea', true);
$order->delete();
Mage::unregister('isSecureArea');